pep-0557.rst

```diff
@@ -739,7 +739,7 @@ also execute code immmediately after constructing the object) is
 functionally equivalent to being able to pass parameters to a
 ``__post_init__`` function.
 
-With ``InitVar``'s, ``__post_init__`` functions can now take
```

Comment:
Isn't that actually the correct one, and the error is really a few lines down where an apostrophe is missing? I think it reads better without the apostrophes, but that's apparently a markup error.

Reply:
Re-reading it again, I also think it is better without the apostrophe. I added a backslash instead, hoping this will fix the markup. What do you think?